package.json

```diff
@@ -23,13 +23,15 @@
   "jest": {
     "coveragePathIgnorePatterns": [
       "src/server/database"
-    ]
+    ],
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true
```

Comment:
I’d rather we don’t do that, or it will collect coverage everytime we run tests, which makes it slower.
We can simply pass `npm run test unit -- --coverage && npx codecov -t ${CODECOV_TOKEN}` to Travis.